src/main/java/org/jenkinsci/plugins/fodupload/SharedPollingBuildStep.java

```diff
@@ -253,7 +253,12 @@ public void perform(Run<?, ?> run,
 
                 // if the polling fails, crash the build
                 if (!result.isPollingSuccessful()) {
-                    run.setResult(Result.FAILURE);
+                    if(result.isScanInProgress()) {

```

Comment:
I might not fully understand what's going on here.  Should we check to see if the new warning option was selected in addition to the in progress scan check, or are we not concerned with that in this context?